documentapi/src/main/java/com/yahoo/documentapi/SyncSession.java

```diff
@@ -6,6 +6,7 @@
 import com.yahoo.document.DocumentPut;
 import com.yahoo.document.DocumentRemove;
 import com.yahoo.document.DocumentUpdate;
+import com.yahoo.document.TestAndSetCondition;
```

Comment:
Is it preferable to use an explicit import or to use a fully qualified class name in the JavaDoc instead?

Reply:
I have never discussed this with anyone, but I prefer it this way.